frontend/src/types.ts

```diff
@@ -563,6 +563,7 @@ export interface TeamType extends TeamBasicType {
     live_events_token: string
     cookieless_server_hash_mode?: CookielessServerHashMode
     human_friendly_comparison_periods: boolean
+    revenue_tracking_config: RevenueTrackingConfig
```

Comment:
Any reason to wrap this in a config with a nested `events` key rather than just exposing the array here? Future-proofness, I suppose? But what could we add here besides events?

Reply:
In the future there might be config about pulling revenue out of the data warehouse. So, future-proofing, yes